src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,9 @@
-const numberToReversedDigits = (number) => {};
+function numberToReversedDigits(number) {
+const array = number.split("");
+const numArray = array.map((element) => Number(element))
+return numArray.reverse()
+}
+
+
```

Comment:
This works, but you can just chain the methods together rather than creating the extra variables:
```
function numberToReversedDigits(number) {
  return number
    .split("")
    .map((element) => Number(element))
    .reverse();
}
```